test/new_relic/agent/error_trace_aggregator_test.rb

```diff
@@ -139,7 +139,7 @@ def test_queue_overflow
         end
 
         errors = error_trace_aggregator.harvest!
-        assert errors.length == max_q_length
+        assert_equal(errors.length, max_q_length)
```

Comment:
I think we want `max_q_length` to be the first argument since it represents the constant and the object under test is `errors.length`

Reply:
Thank you Kayla. [Fixed](https://github.com/newrelic/newrelic-ruby-agent/pull/1482/files#diff-7eeb0fc1575152bf52f0f8a86d1988c691f3228f638155cc55217f336a68e0b3R142)!